src/components/SelectedTableNotification.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { FC } from "react";
+import { Button, Notification } from "@canonical/react-components";
+
+interface Props {
+  totalCount: number;
+  filteredNames: string[];
+  itemName: string;
+  parentName: string;
+  selectedNames: string[];
+  setSelectedNames: (val: string[]) => void;
+}
+
+const SelectedTableNotification: FC<Props> = ({
+  totalCount,
+  filteredNames,
+  itemName,
+  parentName,
+  selectedNames,
+  setSelectedNames,
+}: Props) => {
+  const isAllSelected =
+    selectedNames.length === filteredNames.length && selectedNames.length > 0;
```

Comment:
Good idea to relax the check here, I'll change that.

We need to check in the parent, because in the pagination component we would still receive a component even if the component renders null. But in the pagination component, we want to show the amount of items, when nothing is selected. Thus, we want to receive a null in the no selection case.